R/create_profile_plot.R

```diff
@@ -40,10 +37,11 @@ create_profile_plot <- function(input, output, session,
                                 in_seq, in_taxa,
                                 rank_select, in_select,
                                 taxon_highlight, gene_highlight,
-                                width, height,
-                                x_axis,
-                                type_profile) {
-  # data for heatmap -----------------------------------------------------------
+                                # width, height,
```

Comment:
are these commented out by design? and if so, why not delete them? :joy:

Reply:
So that you habe something to comment 😜 Have removed them all 😄